canvasapi/course.py

```diff
@@ -1410,17 +1412,19 @@ def get_multiple_submissions(self, **kwargs):
         :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
             :class:`canvasapi.submission.Submission`
         """
-        if 'grouped' in kwargs:
-            warnings.warn('The `grouped` parameter must be empty. Removing kwarg `grouped`.')
-            del kwargs['grouped']
+
+        if "grouped" in kwargs and normalize_bool(kwargs["grouped"], "grouped"):
```

Comment:
This might be more easily read as:

```
grouped = kwargs.get("grouped")

if normalize_bool(grouped, "grouped"):
    cls = GroupedSubmission
else:
    cls = Submission
```

Reply:
That is not logically equivalent. If there is no `grouped` value passed, then `kwargs.get("grouped")` will return `None`. 

Then, `normalize_bool(grouped, "grouped")` will throw an exception.

Reply:
Two options, then:

`grouped = kwargs.get("grouped", False)` OR

```
grouped = kwargs.get("grouped")

try:
    if normalize_bool(grouped, "grouped"):
        cls = GroupedSubmission
    else:
         cls = Submission
except ValueError:
    cls = Submission
```